data/911/index.html

```diff
@@ -17,33 +17,37 @@
   downloads:
     - url: https://drive.google.com/a/utah.gov/uc?id=1Sp6v4FmNV5fhdfEWPTxPGHMbnz-yeY47&export=download
       label: 'UtahNG911GIS: File Geodatabase'
-  updates:
-    - May 20, 2020
-    - April 08, 2020
-    - February 28, 2020
-    - February 7, 2020
-    - January 2, 2020
-    - October 9, 2019
-    - September 5, 2019
-    - August 8, 2019
-    - July 10, 2019
-    - June 5, 2019
-    - May 8, 2019
-    - April 2, 2019
-    - February 23, 2019
-    - January 24, 2019
-    - November 29, 2018
-    - October 11, 2018
-    - August 14, 2018
-    - June 27, 2018
-    - May 30, 2018
-    - April 12, 2018
-    - March 16, 2018
-    - February 14, 2018
-    - February 6, 2018
-    - January 19, 2018
-    - January 04, 2018
-    - December 18, 2017
+  updated:
+    - June 24, 2020
+    <details>
```

Comment:
**thought**: i think this should work as it's an html5 feature

Reply:
i guess it bombed.  maybe b/c it's part of a list item?  ... but, no biggie.  i'm also thinking of removing all the previous dates.  the only reason i leave them is to show we've been updating the ng911 db monthly for X amount of time.

Reply:
Ohhh yes, because this is in the front matter. I think you should update the include to add this as a feature! What do you think?

Reply:
front matter... that's the term i was reaching for.  i bet you're right.  i just axed it in my past commit.  it's kind of overkill.  but, i like the idea of the collapsible list.  i'll probably use it at one point.  thanks for the feedback and troubleshooting.

Reply:
also, could have bombed b/c i changed the include keyword (for lack of a better term) from "updates" to "updated".  maybe i'll mess with more and use it as a feature on something else when i have more time.